input/docs/handbook/dependency-inversion/custom-dependency-inversion.md

```diff
@@ -100,16 +112,42 @@ public class AutofacDependencyRegistrar
 }
 ```
 
-
 ## Set the Locator.Current to your implementation
 
 ```csharp
 var resolver = new AutofacDependencyResolver(container);
-// These Initialize methods will add ReactiveUI platform registrations to your container
-// They MUST be present if you override the default Locator
-resolver.InitializeSplat();
-resolver.InitializeReactiveUI();
+// Assigning your custom resolver to Locator.Current will automatically take care of
+// adding all default Splat and ReactiveUI registrations to your container.
 Locator.Current = resolver;
 ```
 
 From this point on calls `Locator.Current` will go against your custom implementation!
+
+## Bonus
```

Comment:
Instead of calling this a bonus, which implies that it is an extra added bit to the auto fac code above.  We should either move this to another page, or explicitly call out using a header tag that this is a different container.